dcrjson/chainsvrwsntfns.go

```diff
@@ -73,6 +73,50 @@ type ReorganizationNtfn struct {
 	NewHeight int32  `json:"newheight"`
 }
 
+// NotifyWinningTicketsCmd is a type handling custom marshaling and
```

Comment:
These should all move into `chainsvrwscmds.go` instead of here in `chainsvrwsntfns.go`.

Notice that in this file the `flags := UFWebsocketOnly | UFNotification` which says these are notifications.  They aren't notifications, rather requests for notifications (aka commands).